src/components/Person/Person.jsx

```diff
@@ -1 +1,15 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+
+export const Person = ({ person }) => {
+  const partner = person.sex === 'm' ? 'wife' : 'husband';
```

Comment:
```suggestion
  const partner = person.sex === 'm' 
    ? 'wife' 
    : 'husband';
```
This formatting will increase readability
![image](https://github.com/mate-academy/react_person/assets/96409563/8518cd5b-6f0f-45f8-9a5e-e175190d8040)